### ISSUE TITLE
Change button configuration for zeemote (or allow custom mapping)

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#2359]
Originally reported on Google Code with ID 1457

```
I think it would be nice to make it so that the user can make what buttons they want
to cause what options they want.

For example, I find it is annoying having to take my finger from the stick to the D
button, to display an answer. Surely it is much more ergonomic to make it so the user
does not have to take their finger off the stick for each and every answer...
What I recommend is to at least swap the A and D buttons, the A button is much easier
to press, and with less effort. Pressing A at the moment plays audio. Not every deck
has audio, but every deck has an answer, I don't see why playing the audio should 'benefit'
from the larger, more accessible, better (in my opinion) button.

Combined with this, once the answer is displayed, having the A button trigger the recommended
answer would be fantastic. Just like pressing space in desktop anki. The recommended
answer exists in the code, I have made use of it personally in the beta 19 source.

Basically I think it would be good to swap button A and D in the reviewer, and to also
let button A trigger the recommended answer. (Just how space acts in anki desktop,
displaying the answer AND answering with the recommended choice)
That, or, allowing us to edit the configuration of each zeemote button for ourselves,
which would be much more work, which I don't think is as important.

Thank you very much :)
```

Reported by `woof.addict` on 2012-10-30 11:10:19
